### PR TITLE
chore: update libp2p-pubsub to 0.0.4

### PR DESCRIPTION
**Pull request by wemeetagain**
Replace local utils with corresponding pubsub imported utils.

### CODE REVIEW

File: test/utils.spec.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 const expect = require('chai').expect
```
Comment (vasco-santos) [inline test/utils.spec.js]: These tests are also in `libp2p-pubsub`, can you remove this file?

**Comment by vasco-santos** [comment]
@wemeetagain I just released `0.0.4` for exporting pubsub utils [libp2p/js-libp2p-pubsub#15](https://github.com/libp2p/js-libp2p-pubsub/pull/15). Can you change the import here to use this export instead? Otherwise, browsers CI will fail.

**Comment by vasco-santos** [comment]
Thanks for the PR @wemeetagain . FYI, I rebased your PR and renamed the commits, in order to pass the commitlint, according to the conventions.

**Comment by wemeetagain** [comment]
:disappointed: we missed a topicCID in the README

**Comment by vasco-santos** [comment]
oh, I will change it! Thanks for reporting it

**Pull request merged**